script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt

```diff
@@ -751,45 +852,49 @@ Senpai's hand brushes against mine.
 [sha:0aae74c6d51799a9108f36c3804a40b5d53b898d]{
 -- Page 145, Offset 38736.
 -- ぶんぶん、と元気よく振って、離した。
-After a vigorous handshake, we finally part ways.
+After a firm handshake, we pull apart.
 }
 [sha:204f83ab7bde645ecffd26f672c8093079cfaeed]{
 -- Page 146, Offset 38737.
 -- 「遠野くんは校舎から出ないように。
-"Try not to go out of the building this time.
+"Please don't leave the school building.
 }
 [sha:cbd5771269a08ec3b2d2cb0a7d72c0f9649655a6]{
 -- Page 146, Offset 38738.
 -- あ、でも危険だと判断したら、校庭か体育館に移動してください」
-Ah, but if you think you're in danger, then please move to either the school yard or the gym."
+Ah, unless you think you're in danger,
+ in which case please evacuate to either the schoolyard or the gym."
 }
 [sha:d1e607bec1aeddf47516bfe02b96d28660a2744f]{
 -- Page 147, Offset 38739.
 -- 青い後ろ姿が階段を駆け上がっていく。
-Her blue figure runs off toward the stairs.
+Her blue figure disappears up the stairs.
 }
 [sha:cced75ff595d48b289febbe8a5430595f9e4904d]{
 -- Page 147, Offset 38740.
 -- 先輩は今度こそ行ってしまった。
-With that, Senpai is gone for good.
+// As opposed to the false start a couple lines up I guess
+She actually went and left this time.
 }
 [sha:ed5d171477f87a3655854ea752c82fc11520d49a]{
 -- Page 148, Offset 38741.
 -- ―――遠く、今はもう変わり果てた市街地から地響きが聞こえてくる。
-―――Far off in the distance, I hear rumblings coming from the urban area that's already completely changed.
+// 地響き seems to be like subterranean/earth related, so rendering as low-pitched
+―――Far off in the distance,
+ I hear a low-pitched rumble emanating from the now unrecognizable city.
 }
 [sha:8c09898a0437f1845cb0ba8578672150173f59a0]{
 -- Page 148, Offset 38742.
 -- それは大気の軋む音、世界の悲鳴だ。
-It is the sound of the creaking atmosphere; the cries of the world.
+It is the sound of the atmosphere creaking――of the world crying out.
 }
 [sha:700131f68cf03c575895aa2ed6505e5a079e4f94]{
 -- Page 148, Offset 38743.
 -- 天変地異そのものと言える白い吸血姫。
-A white vampire princess that is the very definition of a cataclysm.
+The vampire princess in white is the very definition of a cataclysm.
 }
 [sha:95fd45ceac324f6344ae5d34232f1218e04f996b]{
 -- Page 148, Offset 38744.
 -- それを正面から打倒する為、彼女は戦場へ赴いた。
-In order to defeat it head-on, Senpai heads toward the battlefield.
+In order to face and defeat her, Senpai rushes headlong toward the battlefield.
```

Comment:
Would rather bind these together.

>Faced with the cataclysmic vampire princess in white———

>Senpai rushes headlong onto the battlefield, intent on defeating her in straight battle.

Reply:
Hmm, first one makes it sound like she's already here.

Reply:
It's a metaphorical face